app/src/test/java/org/oppia/app/topic/revisioncard/RevisionCardActivityLocalTest.kt

```diff
@@ -0,0 +1,129 @@
+package org.oppia.app.topic.revisioncard
+
+import android.app.Application
+import androidx.appcompat.app.AppCompatActivity
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import javax.inject.Inject
```

Comment:
imports which are of `java, javax, kotlin` will come at the end of the import list. 

```
import javax.inject.Inject
import javax.inject.Singleton
```

This can be achieved with the `optimise imports` in the menu toolbar. The only thing we have to care about while using `optimise imports` is that after doing it some imports might get wildcard imports and we don't want wildcard imports. 

To achieve `reformat code and optimise imports` together with one shortcut you can have a look [here](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros). Let me know if you find any difficulty in this, happy to help.

Reply:
Done.